Makefile

```diff
@@ -158,6 +164,7 @@ bundle: manifests kustomize
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
 	$(KUSTOMIZE) build config/manifests | $(OPERATOR_SDK) generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
 	$(OPERATOR_SDK) bundle validate ./bundle
+	cd config/manager && $(KUSTOMIZE) edit set image controller="discovery-operator:latest"
```

Comment:
Should this be hardcoded or should it be - `$(IMG):$(VERSION)`?

Reply:
Yes, the edit after the command is a convenience command to reset the Kustomize file to its default value after running the command. That way you don't have to reset the file or accidentally commit your custom image/version when doing Git commits.